website/docs/d/disk_encryption_set.html.markdown

```diff
@@ -0,0 +1,66 @@
+---
+subcategory: "Compute"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_disk_encryption_set"
+sidebar_current: "docs-azurerm-datasource-disk-encryption-set"
+description: |-
+  Gets information about an existing Disk Encryption Set
+---
+
+# Data Source: azurerm_disk_encryption_set
+
+Use this data source to access information about an existing Disk Encryption Set.
+
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Disk Encryption Set exists.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Disk Encryption Set exists.
+
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - Resource Id

```

Comment:
can we update this to use the terminology used for `id` in every other resource?

Reply:
No problem